src/templates/static/mt5_signals.jsx

```diff
@@ -0,0 +1,240 @@
+                                    {it.L('If you are a professional trader, the MT5 trading signals service allows you to share your strategies with other traders for free or a subscription fee that you determine. When traders subscribe to your signal, your deals are automatically replicated on their accounts each time you place a trade.')}
+                                </p>
+                            </div>
+
+                            <div className='fill-bg-color gr-padding-30 fullwidth' >
+                                <div className='container gr-padding-30 gr-child'>
+                                    <h2 className='center-text'>
+                                        {it.L('Benefits of being an MT5 provider')}
+                                    </h2>
+                                    <div className='fill-bg-color gr-padding-30'>
+                                        <div className='container section'>
+                                            <div className='gr-padding-10 facts'>
+                                                <div className='gr-row no-padding'>
+                                                    <Box>
+                                                        <BoxInner className='border-right-top' image='easy_setup' text={it.L('Easy to set up – no installation required.')} />
+                                                        <BoxInner className='border-right-top' image='additional_income' text={it.L('Additional income stream from monthly subscriptions')} />
+                                                        <BoxInner className='border-right-top' image='automated_copy_trade' text={it.L('Automated copying – no extra work from you')} />
+                                                    </Box>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div className='gr-padding-30' >
+                                <div className='container gr-padding-30 gr-child'>
+                                    <h2 className='center-text'>
+                                        {it.L('How to register as a signal provider')}
+                                    </h2>
+                                    <div className='gr-row gr-padding-30'>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/signal_provider_register_1.png')} />
+                                            <ol>
+                                                <li>{it.L('Go to the MQL5 signals showcase page and click on the <strong>Create signal</strong> button.')}</li>
```

Comment:
it would be better to use placeholder for the `<strong>` tags, so translators don't accidentally break the tags.